tests/solidity_files/output/ERC721TokenReceiver.abi

```diff
@@ -0,0 +1 @@
+[{"inputs":[{"internalType":"address","name":"","type":"address"},{"internalType":"address","name":"","type":"address"},{"internalType":"uint256","name":"","type":"uint256"},{"internalType":"bytes","name":"","type":"bytes"}],"name":"onERC721Received","outputs":[{"internalType":"bytes4","name":"","type":"bytes4"}],"stateMutability":"nonpayable","type":"function"}]
```

Comment:
Why is this file compiled and not ERC721.bin / ERC721.abi ?

Reply:
? the ERC721.sol indeed defined the ERC721 and the ERC721TokenReceiver, this makes a total of 3 contracts compiles, as found in the `output` folder

Reply:
but maybe we should just remove the `.abi` output as we only plan to use the `.bin`. The `*.opcode` may still be useful for debugging purposes

Reply:
yes but it says ERC721.bin is empty on my screen

![image](https://user-images.githubusercontent.com/66871571/201972388-c1bef585-54f4-4c3a-958a-66ad664305f0.png)